support/fileutil/io_go1.16.go

```diff
@@ -0,0 +1,14 @@
+//go:build go1.16
+// +build go1.16
+
+package fileutil
+
+import "os"
+
+func ReadFile(file string) ([]byte, error) {
+	return os.ReadFile(file)
+}
+
+func WriteFile(file string, data []byte) error {
+	return os.WriteFile(file, data, 0755)
```

Comment:
You can use recommended octal notation

It's available since Go 1.13

https://go.dev/ref/spec#Go_1.13


```suggestion
	return os.WriteFile(file, data, 0o755)
```

Reply:
Haha, such interesting feature

Reply:
I discovered when I started using gofumpt.

https://github.com/mvdan/gofumpt

It can be easily added to gopls, or any IDE.

Reply:
nice refer link

Reply:
Gofumpt is developed by the same guy who made shfmt another tool, I hope you heard about

https://github.com/mvdan/sh

It changed my life when working on script.

To be used with or without https://github.com/koalaman/shellcheck